ansible/deploy-backend.yml

```
@@ -0,0 +1,148 @@
+  become: yes
+  gather_facts: true
+  
+  vars:
+    service_account_file: gcp_client_hsecret.json
+    ansible_ssh_common_args: '-i ~/.ssh/google_compute_engine -J devops@bastion.us-central1-c.teachua-6147'
+    backend_port: 8080
+    db_host: "34.27.120.127"
+    db_name: "teachua"
+    db_user: "teachua_user"
+    db_password: "securepassword123"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)